articles/anomaly-detection/passwords-checking.md

```diff
@@ -0,0 +1,229 @@
+
+If it works, you will get a response similar to the following one:
+
+```js
+{
+    "id":"job_abcdef1234567890",
+    "status":"pending",
+    "type":"passwords_checking",
+    "connection":"abcd"
+}
+```
+
+The returned entity represents the passwords checking job. At this point the job is queued on our system and will be started
+as soon as possible. Once the job is finishes you will get a log entry `fpc` (if failed) or `spc` (if succeed). You can query the logs through [Management API2](/api/management/v2) or use our [Dashboard](${manage_url}/#/logs) to get them. You can also query job status anytime you want using the appropriate endpoint (see bellow).
+
+### Querying job status
+Once a job has been started you can query its status whenever you want, to do so simply make a `GET` request to `/api/jobs/{id}`, the `id` is the one that has been handed to you when you started the job (see above). You will get a response similar to
+
+```js
+{
+    "id":"job_abcdef1234567890",
+    "status":"pending",
+    "type":"passwords_checking",
+}
+```
+
+The `status` field might contain any of the following values:
+
+- pending: The job is still queued or being executed on our system. Wait some minutes and query again to know when it has finished.
+- completed: The job is complete you can query the errors and the results (see bellow).
+- failed: The job has failed, you must query the errors and check your logs to see what went wrong.
+
+### Querying results
+Once the job has finished you are ready to gets it results and act based on them. To get the results you should make a request to `/api/{job_id}/results`. You will get a JSON Array response, similar to:
+
+```js
+[
+  {
+    "email": "john.doe1@example.com",
+    "status": "password_matched"
+  },
+  {
+    "username": "john.doe2",
+    "status": "invalid_password"
+  },
+  {
+    "username": "john.doe2",
+    "status": "user_not_found"
+  }
+]
+```
+
+- Email or username will be available depending the information you provided as input.
+- Status might have any of the following values:
+  * password_matched: the password you provided for that user matched
+  * invalid_password: the password you provided haven't matched
+  * user_not_found: the user wasn't available in the connection database you specified.
+
+Keep in mind that we try to recover from most of the errors instead of failing the job, so even when the job is complete
+there might be rows that have had errors and so they are not included in the results. You should check the errors (see below) to get more information about what have happened.
+
+```
+*IMPORTANT:* To keep a high security standard you must delete the job once you have the results and store them securely. The results will be automatically deleted after some time of completing / failing the job. Once the job has been processed the input files are deleted automatically and you will be not able to access them.
```

Comment:
+*IMPORTANT:* For security, you should delete the job once you have retrieved your results. The results will be automatically deleted after a period of time, but it is a good practice to clean out the data as soon as possible regardless.